reviews/models/mixins.py

```diff
@@ -244,30 +245,15 @@ def get_context(self):
 
 # Handle email notifications including: update comment, accept, and reject of submission.
 @reviews_signals.reviews_email.connect
-def reviews_notification(self, context):
-    timestamp = timezone.now()
-    event_type = utils.find_subscription_type('global_reviews')
-    template = context['template'] + '.html.mako'
-    for user_id in context['email_recipients']:
-        user = OSFUser.load(user_id)
-        subscriptions = get_user_subscriptions(user, event_type)
-        for notification_type in subscriptions:
-            check_user_subscribe = subscriptions[notification_type] and user_id in subscriptions[notification_type] and notification_type != 'none'  # check if user is subscribed to this type of notifications
-            if check_user_subscribe:
-                node_lineage_ids = get_node_lineage(context.get('reviewable').node) if context.get('reviewable').node else []
-                context['user'] = user
-                context['no_future_emails'] = notification_type == 'none'
-                send_type = notification_type if notification_type != 'none' else 'email_transactional'
-                message = mails.render_message(template, **context)
-                digest = NotificationDigest(
-                    user=user,
-                    timestamp=timestamp,
-                    send_type=send_type,
-                    event='global_reviews',
-                    message=message,
-                    node_lineage=node_lineage_ids
-                )
-                digest.save()
+def reviews_notification(self, context, node):
+    user = context['email_sender']
+    auth = Auth(user)
+    time_now = timezone.now()
+    email_recipients = context['email_recipients']
+    email_recipients.remove(user._id)  # remove email sender
+    for user_id in email_recipients:
+        context['target_user'] = OSFUser.load(user_id)
```

Comment:
In `notify` it says `target_user` is "The user who the current comment is a reply to", which doesn't seem right for our use. Seems like we want to pass as `user` the user who triggered this transition (the moderator who made a decision, or the user who resubmitted a preprint), which is available in the transition callback as `ev.kwargs['user']`

Reply:
This condition was confusing me in notify method:
    if target_user and event_type in constants.USER_SUBSCRIPTIONS_AVAILABLE:
        # global user
        subscriptions = get_user_subscriptions(target_user, event_type)
    else:
        # local project user
        subscriptions = compile_subscriptions(node, event_type, event)
it seems that nofiy get subscriptions global if target_user has value, since we use now using global subscriptions not local project one, I pass the recepient as a target user in each loop.